storage/innobase/xtrabackup/test/t/bug1707918.sh

```diff
@@ -0,0 +1,27 @@
+
+master_id=1
+slave_id=2
+
+start_server_with_id $master_id
+load_sakila
```

Comment:
We don't need to load sakila database for this test. We can test on much smaller dataset and save some precious Jenkins time :) Even further, I think this test can be combined with `ib_slave_info.sh`, we can make streaming backup there along with local backup.